apps/epic-react/src/pages/modules/[module]/completed.tsx

```diff
@@ -0,0 +1,167 @@
+        }}
+      >
+        <Party />
+        <div className="flex h-full min-h-screen w-full items-center justify-center p-8 pt-20 text-text sm:pt-0 lg:p-0">
+          <div className="mx-auto grid max-w-screen-lg grid-cols-1 items-center justify-center gap-16 md:grid-cols-5">
+            <motion.div
+              initial={{scale: 0.6}}
+              animate={{scale: 1}}
+              transition={{
+                duration: 1.6,
+                type: 'spring',
+                mass: 1.5,
+                stiffness: 20,
+              }}
+              className="relative col-span-1 mx-auto flex h-64 w-64 items-center justify-center overflow-hidden rounded-full bg-gray-100 p-16 md:col-span-2 lg:h-80 lg:w-80"
+            >
+              <motion.img
+                initial={{y: 300}}
+                animate={{y: 0}}
+                transition={{
+                  duration: 2.5,
+                  type: 'spring',
+                  mass: 1.8,
+                  stiffness: 50,
+                }}
+                src={
+                  module
+                    ? module?.image ||
+                      'https://p-ZmFjNlQ.b3.n0.cdn.getcloudapp.com/items/DOuAzNP9/ER_Build4_Launch@2x.png'
+                    : 'https://p-ZmFjNlQ.b3.n0.cdn.getcloudapp.com/items/DOuAzNP9/ER_Build4_Launch@2x.png'
+                }
+              />
+            </motion.div>
+            <div className="flex flex-col items-center md:col-span-3 md:items-start">
+              <h1 className="mb-4 inline-flex max-w-lg items-center text-2xl font-semibold leading-tight sm:text-4xl">
+                <motion.svg
+                  className="mr-2 text-indigo-500"
+                  width="32"
+                  height="32"
+                  viewBox="0 0 32 32"
+                  initial={{scale: 0}}
+                  animate={{scale: 1}}
+                  transition={{duration: 4, type: 'spring'}}
+                >
+                  {/* <g fill="none" fillRule="evenodd"> */}
+                  <circle cx="16" cy="16" r="16" fill="#6875F5" />
+                  <motion.path
+                    initial={{pathLength: 0}}
+                    animate={{pathLength: 1}}
+                    transition={{duration: 1, ease: 'easeOut'}}
+                    stroke="#FFF"
+                    fill="none"
+                    strokeDasharray="0 1"
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    strokeWidth="3"
+                    d="M8 16 13 21 24 11"
+                  />
+                  {/* </g> */}
+                </motion.svg>
+                Great work!
+              </h1>
+              {module && (
+                <p className="mb-8 text-center text-base  text-gray-600 sm:text-lg md:text-left">
+                  You&apos;ve completed {module.title} workshop.
+                </p>
+              )}
+              {module && <ModuleCertificate module={module} />}
```

Comment:
using current skill implementation of module certificate as the logic was already in here vs porting over the epic-react-gatsby implementation